cpovc_forms/models.py

```diff
@@ -1053,7 +1053,7 @@ def __unicode__(self):
 
 
 class OVCMonitoring(models.Model):
-    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    monitoring_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
@thomasbshop , please confirm this model change as it may affect migrations and other code